pages/mod/_id/settings.vue

```diff
@@ -497,12 +497,30 @@ export default {
 
     .main-info {
       margin-bottom: var(--spacing-card-lg);
+
```

Comment:
NYF but the adding team members thing overflows on mobile
![image](https://i.imgur.com/LpBihzb.png)